schema/event_mapping_entity.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$udmi_version": "1.4.0",
+  "title": "Mapping Event Entity",
+  "type": "object",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "additionalProperties": false,
+  "properties": {
+    "code": {
+      "description": "",
+      "type": "string"
```

Comment:
type should be an enum

Reply:
I don't believe so, this field will receive values like `"US-SVL-MP2"`

Reply:
hmmm, ok -- I think we'll have to follow-up with this team about this because that definition seems to be a mess... "US-SVL-MP2" is not a type.